source/slang/slang-reflection-api.cpp

```diff
@@ -2033,6 +2140,25 @@ SLANG_API SlangInt spReflectionTypeLayout_getSubObjectRangeSpaceOffset(
     return extTypeLayout->m_subObjectRanges[subObjectRangeIndex].spaceOffset;
 }
 
+SLANG_API SlangReflectionVariableLayout* spReflectionTypeLayout_getSubObjectRangeOffset(
```

Comment:
Should it be named getSubObjectRangeVarLayout?

Reply:
I suppose it could. It's a little tricky because it doesn't really represent the entire range so much as the "leaf" variable where the range starts.

I guess there's actually one high-order-bit design thing here that could impact the entire "flat" reflection API: it is possible that we should just have every binding range support a `getBindingRangeLeafVarLayout()` to match the `getBindingRangeLeafTypeLayout()` and at that point we realistically don't need the have a separate reflection setup for "descriptor ranges" at all.